### ISSUE TITLE
Setting for Ignored names

### ISSUE THREAD

**Issue by zyy7259** [atom/find-and-replace#913]
Can we have a setting for `Ignored names`?

I know there is a global setting for the `Ignored names`. But sometimes I just want some specific folders to be excluded from `find-and-replace`. I found the package `fuzzy-find` have this setting. Can we have one?

**Comment by TSMMark** [comment]
Yes `fuzzy-find` has this and it allows me to ignore `node_modules/*` which is fantastic. Thanks

**Comment by jorgeorpinel** [comment]
**Ignored Names** in Atom Core Settings works for find-and-replace as well.
